docs/readmes/orc8r/deploy_build.md

```diff
@@ -77,14 +77,20 @@ To start, create a private GitHub repo to use as your Helm chart repo. We'll
 refer to this as `GITHUB_REPO`.
 
 Next, package the Magma Helm charts and publish them to the GitHub
-repo
+repo. You'll have to check out a temporary commit to build the `1.4.36` version
+of the orc8r chart for 1.3.x because `1.4.37` on the head of the release
+branch has some changes that you shouldn't try to deploy yet.
```

Comment:
What's up with the 1.4.36 vs. 1.4.37 messiness?

In any case, feels cleaner to put it as a `NOTE: if you're trying to package 1.4.37 ...`

Reply:
Nobody should be trying to package 37 right now except us